internal/collector/patroni.go

```diff
@@ -168,14 +168,22 @@ func EnablePatroniMetrics(ctx context.Context,
 			},
 		}
 
+		// If there are exporters to be added to the metrics pipelines defined
+		// in the spec, add them to the pipeline.
+		exporters := []ComponentID{Prometheus}
```

Comment:
So there's no way for a user to remove the Prometheus exporter? (Of course, if they did, we'd maybe also want to change the way we expose the port?)

Reply:
I keep thinking about the way we deal with debug for logs as a model, but maybe that's too different?

Reply:
Yeah feels pretty different to me. The debug exporter is there just so that there is some way to view logs if the user doesn't provide another solution, but the Prometheus exporter is key for integrating into our monitoring stack.

Reply:
Yeah, agreed -- we can use this PR to add other exporters and maybe another to remove Prometheus/remove the exposed port (if we want to. I can imagine someone who wants their own exporter might not want Prometheus at all).

(Though also I'm still curious if there's other ports to expose or other changes for other metrics exporters, but at least we've covered the major ones, I think.)